src/WebJobs.Script/Binding/FunctionBinding.cs

```diff
@@ -221,6 +222,12 @@ internal static async Task BindStringAsync(BindingContext context)
             context.Value = str;
         }
 
+        internal static async Task BindParameterBindingData(BindingContext context)
+        {
+            var parameterBindingData = await context.Binder.BindAsync<ParameterBindingData>(context.Attributes);
+            context.Value = parameterBindingData;
+        }
```

Comment:
```suggestion
        internal static Task BindParameterBindingData(BindingContext context)
        {
            return context.Binder.BindAsync<ParameterBindingData>(context.Attributes);
        }
```

Reply:
This breaks the workflow `context.Value` needs to be set